manifest.json

```diff
@@ -5095,6 +5095,52 @@
                 }
             }
         },
+        "me.badhaloninja.AvatarCreatorUtils": {
+            "name": "AvatarCreatorUtils",
+            "description": "Adds various options to the avatar creator",
```

Comment:
Probably too vague? Could include an example or two here so those reading it don't have to dig into the source to know atleast some amount to expect.